sass/components/_files.scss

```diff
@@ -306,8 +306,10 @@
 
                     &:active {
                         background: rgba(var(--button-bg-rgb), 0.08);
-                        color: v(button-bg);
-                        fill: v(button-bg);
+
+                        .icon {
+                            color: v(button-bg);
```

Comment:
the usage of `v()` is deprecated in the webapp. Could you please replace that with `var(--button-bg)`? Thanks!